samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Application.java

```diff
@@ -19,8 +19,9 @@
 import com.microsoft.bot.integration.SkillHttpClient;
 import com.microsoft.bot.integration.spring.BotController;
 import com.microsoft.bot.integration.spring.BotDependencyConfiguration;
+import com.microsoft.bot.sample.dialogrootbot.Bots.RootBot;
```

Comment:
I'm not sure to add these changes in the PR, we are not fixing the DBI, we need to rollback this